cwf/gateway/docker/python/Dockerfile

```diff
@@ -50,17 +53,17 @@ RUN apt-key add /tmp/jfrog.pub && \
     apt-add-repository "deb https://facebookconnectivity.jfrog.io/artifactory/list/dev/ bionic main" && \
```

Comment:
We should start pushing as distribution=focal and change s/bionic/focal here

Reply:
Currently there is no such release so this cause failure in build process.